src/components/button/Button.js

```diff
@@ -0,0 +1,24 @@
+import './button.scss';
+
+export class Button extends React.Component {
+  constructor() {
+    super();
+    this.state = { btn: '' };
+  }
+
+  changeClass() {
+    if (this.state.btn === 'active') {
+      this.setState({ btn: '' });
+    } else {
+      this.setState({ btn: 'active' });
+    }
```

Comment:
ok, leave it as it is. But next time is better to use _btn_ as boolean flag _true/false_ and use `this.setState({ btn: !this.state.btn });`